digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/process/TrackSectionOrder.scala

```diff
@@ -59,6 +59,15 @@ object TrackSectionOrder {
   }
 
   def mValueRoundabout(seq: Seq[ProjectLink]): Seq[ProjectLink] = {
+    def adjust(pl: ProjectLink, sideCode: Option[SideCode] = None, startAddrMValue: Option[Long] = None,
+               endAddrMValue: Option[Long] = None, startCalibrationPoint: Option[Option[CalibrationPoint]] = None,
```

Comment:
was Option[Option[CalibrationPoint]] intended?

Reply:
Yes. I'll add a comment for it.
For clearing a calibration point, Some(None) is used.
For not clearing, None is used.